<launch>

	<!-- START A GROUP WITH A NAMESPACE -->
	<group ns="asc">

		<!-- LAUNCH A "I2C for Motors and Servos" NODE -->
		<node
			pkg    = "asclinic_pkg"
			name   = "i2c_for_motors_and_servos"
			output = "screen"
			type   = "i2c_for_motors_and_servos"
		/>

	</group>

</launch>
